### ISSUE TITLE
Linear Map view is showing the sequence map

### ISSUE THREAD

**Issue by XimeMora** [TeselaGen/openVectorEditor#228]
For linear sequences the Linear Map is showing the sequence base-pairs, so basically is showing the same information as the Sequence Map, I think the sequence base-pairs shouldn't be displayed in the Linear Map. This is only happening for linear sequences (I imported one from app.teselagen.com)

![sequence map](https://user-images.githubusercontent.com/21211903/35830187-3b340464-0a7a-11e8-82bc-950ceb541f19.PNG)

![linear map](https://user-images.githubusercontent.com/21211903/35830203-4430bbde-0a7a-11e8-92fc-f223f2664186.PNG)

**Comment by tnrich** [comment]
@XimeMora I disagree on this count, I think that we should show the bps if the linear view is zoomed in enough.
